docs/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,134 @@
+<!--
+Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+
+SPDX-License-Identifier: curl
+-->
+
+# Infrastructure in the curl project
+
+Overview of infrastructure we maintain, host and run in the project for the
+project.
+
+## git repository
+
+Since 2010, the main curl git repository has been hosted by GitHub, available
+at https://GitHub.com/curl/curl.
+
+We also use the issue tracker, pull requests and discussions on GitHub.
+
+curl has an "enterprise account" on GitHub and is an "organization" on the
+site.
+
+We accept sponsorship via Google Sponsors.
```

Comment:
You mean Github?

Randomly seeing this from mastodon :)

Reply:
oh, oops

Reply:
Thanks!